src/main/kotlin/org/vineflower/ijplugin/VineflowerInvoker.kt

```diff
@@ -60,7 +59,7 @@ class VineflowerInvoker(classLoader: ClassLoader) {
         )
 
 
-    private val baseDecompilerClass = classLoader.loadClass("org.jetbrains.java.decompiler.main.decompiler.BaseDecompiler")
+    private val baseDecompilerClass = classLoader.loadClass("org.jetbrains.java.decompiler.main.Fernflower")
```

Comment:
Is this a change in VF's API? Will this work on older version of VF? We need to stay supporting older VF/QF versions. That's what the `Feature`s down below are for.

Also, these variable names need to be changed.

Reply:
Apparently, I didn't look closely enough when trying to work around some NPEs done during debugging and updating